kim/field.py

```diff
@@ -442,6 +442,49 @@ class UserMapper(Mapper):
     marshal_pipeline = DecimalMarshalPipeline
     serialize_pipeline = DecimalSerializePipeline
 
+class FloatFieldOpts(FieldOpts):
```

Comment:
I was thinking on inherit from Decimal.. I'm happy to do a more reusable structure if you want me to.